samples/highcharts/demo/bar-stacked/demo.js

```diff
@@ -1,18 +1,20 @@
-// Data retrieved from: https://www.uefa.com/uefachampionsleague/history/
+// Data retrieved from: https://ferjedatabanken.no/statistikk
 Highcharts.chart('container', {
     chart: {
         type: 'bar'
     },
     title: {
-        text: 'UEFA CL top scorers by season'
+        text: 'Ferry passengers by vehicle-type 2024'
     },
     xAxis: {
-        categories: ['2020/21', '2019/20', '2018/19', '2017/18', '2016/17']
+        categories: [
+            'Mai', 'April', 'Mars', 'Februar', 'Januar'
```

Comment:
Use English month names. 

Also, it looks kind of unintuitive that the months are listed in descending order. Actually I think that the visualization too is better if it starts with January on top. We are used to reading top down.

Reply:
Followed the format of the demo. Will update.